.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules/
+/CamWidget/
```

Comment:
Co je CamWidget? v kodu jsem nic takoveho nenasel..

Reply:
nějaká rozumnější integrace CairoPainter a CairoFrame a nechci aby se to teď do tohohle gitu pletlo
...no vždyť právě